docs/SUMMARY.md

```diff
@@ -51,5 +51,5 @@
   * [Contributing](developers/contributing.md)
 * [Additional Notes](notes/README.md)
   * [Comparison Table](notes/comparison.md)
-  * [Popular Extensions](https://github.com/chartjs/awesome)
```

Comment:
I would keep this link direct. Saves a roundtrip.

Reply:
Agreed, but Gitbook will consider the `extensions.md` page only if it's part of the summary :\

Reply:
Could make a section of old / redirected pages. But I guess that extra roundtrip is ok in this case too.

Reply:
How it would work better to make a "old links" section?

Reply:
Though, I'm not sure we want to maintain old links for everything, especially if we are going to get rid of GitBook at some point. I don't think we need this redirect either (and would keep the direct link), but @stockiNail seems worried about this page not being referenced anymore :)

Reply:
So the "Popular extensions" would point to awesome and somewhere else (under "old links" for example) extensions.md would be listed - just for Gitbook to notice it. I might be wrong how Gitbook works though.